src/library_wasmfs_fetch.js

```diff
@@ -9,12 +9,18 @@ mergeInto(LibraryManager.library, {
   // will fetch() the data from the network asynchronously. Otherwise, after
   // that fetch it behaves just like JSFile (and it reuses the code from there).
 
+  // Contains a mapping of backend files to their file names.
+  $wasmFS$backendFileNames: {},
+
   _wasmfs_create_fetch_backend_js__deps: [
     '$wasmFS$backends',
     '$wasmFS$JSMemoryFiles',
+    '$wasmFS$backendFileNames',
```

Comment:
I don't love the use of `$` in the middle of these identifiers... I wonder if we can switch to using `_`... not related to this PR I guess.. so maybe a followup or a separate prep PR?